docker-compose.yml

```diff
@@ -41,19 +41,31 @@ services:
             - AXIS_CTXAVAILABLETRANSPORTS=http;tchannel
 
             - BEHAVIOR_RAW=client,server,transport
+            - BEHAVIOR_SKIP_RAW=client=java+transport=tchannel,server=java+transport=tchannel
             - BEHAVIOR_JSON=client,server,transport
+            - BEHAVIOR_SKIP_JSON=client=java+transport=tchannel,server=java+transport=tchannel
             - BEHAVIOR_THRIFT=client,server,transport
+            - BEHAVIOR_SKIP_THRIFT=client=java+transport=tchannel,server=java+transport=tchannel
```

Comment:
Yes, I'll update this PR and unpin master after I land https://github.com/crossdock/crossdock/pull/83